pos_safe_box/models/pos_session_validation.py

```diff
@@ -120,9 +120,9 @@ def account_move_vals(self, statement):
             raise ValidationError(_("Account cannot be found for this company"))
         amount = statement.total_entry_encoding
         if amount > 0:
-            statement_account = statement.journal_id.default_credit_account_id
+            statement_account = statement.journal_id.payment_debit_account_id
```

Comment:
must be statement_account = statement.journal_id.default_account_id, you can remove the if and else as it is the same in both cases

Reply:
Done 😊